### DISCUSSION TITLE
Online model merger

### DISCUSSION THREAD

**Discussion by aolko** [civitai/civitai#132] [Ideas]
It would be nice if we could check off the models on-site, merge them and get generated ckpt/safetensors model with an optional new model page

**Comment by maxhulker** [comment]
This is a huge implementation. 
We have plans to offer functionality like this, but it will be later down the line

**Comment by oovm** [comment]
Are there any new developments? For example, allowing to select models in the vault, set specific scales and blend?

**Reply by aolko** [reply]
Let's be honest, it's not yet another coin-based/cosmetic/useless feature, of course it will be at the bottom of their backlog

**Reply by maxhulker** [reply]
We have a couple other priorities first around content discovery, but its still on the roadmap.

**Reply by aolko** [reply]
i'm not surprised to see another excuse
